src/Native/ObjWriter/llvm.patch

```diff
@@ -1,7 +1,7 @@
-diff --git a/include/llvm/MC/MCObjectStreamer.h b/include/llvm/MC/MCObjectStreamer.h
```

Comment:
Why `a` and `b` were changed to `i` and `w`? Could you undo this changes?

Reply:
@alpencolt I got these diffs by the command ```git diff``` I'll look what I can do with this